content/docs/start/experiment-management/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Get Started: Experiment Management'
+description:
+  'Get started with experiment management and visualization in DVC, without
+  bloating Git history.'
+---
+
+# Get Started: Experiment Management
+
+Let's look at DVC's features from the perspective of machine learning experiment
+management. This includes bootstrapping, running and queueing experiments
+(possibly in parallel), as well as visually comparing the results to commit the
+best ones to Git.
```

Comment:
Added this intro to the Exp Mgmt chapter since we'd be using the index page after all.

> Related context in https://github.com/iterative/dvc.org/pull/3956#issuecomment-1253067819.